### DISCUSSION TITLE
multiple icons in one node

### DISCUSSION THREAD

**Discussion by semokhtari** [visjs/vis-network#1890] [Q&A]
Hiiiiiiii
I have a question, thank you for your help:)
How to have multiple icons in one node?

Regards,
S Mokhtari

**Comment by nikhil-varma** [comment]
@semokhtari Maybe you can use the technique here to add multiple icon elements? 
- https://visjs.github.io/vis-network/examples/network/nodeStyles/HTMLInNodes.html
Basically creating an encoded string. Assuming you can overlook the browser support like what it tells in this page.

**Reply by ProIntegritate** [reply]
Related question:

Is there a way to properly add glyphs (icon-icons) to each node _without using inline HTML_ - around the icons, like in this video (I'm using D3JS in the video)
https://www.youtube.com/watch?v=CG1vVBoVtKw

For a looong time, I've been looking for a way to add glyphs like this in VisJS and it would be really useful, like showing status icons on nodes to determine various things, as an example showing the type of host it is (i.e. Windows/Mac/Linux icon)

**Reply by nikhil-varma** [reply]
@ProIntegritate I don't see this being supported natively unless I am missing something. The only native support is font icons and inline, encoded HTML string wrapped with a SVG foreign object tag. I would also prefer to have such support natively but looks like VisJS may not have it.
